bot/controllers/twitchLive.js

```diff
@@ -21,8 +23,10 @@ class StreamerLiveController extends BaseController {
     const twitchName = message.parsed[1];
     const twitchURL = `https://www.twitch.tv/${twitchName}`;
     const twitchGame = message.parsed[2];
-    const streamMessage = `Hey everyone! ${message.author.username} is going live playing ${twitchGame}!! Check him out at ${twitchURL}!`;
-    return streamMessage;
+    const streamMessage = `Hey everyone! ${message.author.username} is going live playing ${twitchGame}!! Check him out at ${twitchURL} !`;
+    const targetChannel = message.guild.channels.get('319548128267206666');
+    const sender = message.author.username;
+    return targetChannel.send('```' + sender + ' has an announcment: ```' + streamMessage);
```

Comment:
App crashes because I do not have the specific channel ID for testing. Make a provision to handle if the channel exists or not, maybe by sending the message to the default channel if the one you are searching for can't be found.

Reply:
Definitely, need to make sure the channel ID is changed appropriately so that it will in any channel not just the one with that specific ID.

Reply:
changed this to look for a specific channel name